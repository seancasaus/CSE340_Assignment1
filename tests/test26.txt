G9MdN (v)|((0)|((I)*)) , tNxlb (((S).(q)).(M)).(n) , zeXdI ((H).(N))|((k).(z)) , r9Bvt ((W).(q)).((L)|((_).(t))) , SYQrI (((a).(W)).(t))* #
"HN0HNvSqMnWqLkzaWtaWtWqL0aWtaWtSqMnHN " 
