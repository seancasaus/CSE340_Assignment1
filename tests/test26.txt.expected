zeXdI , "HN"
G9MdN , "0"
zeXdI , "HN"
G9MdN , "v"
tNxlb , "SqMn"
r9Bvt , "WqL"
zeXdI , "kz"
SYQrI , "aWtaWt"
r9Bvt , "WqL"
G9MdN , "0"
SYQrI , "aWtaWt"
tNxlb , "SqMn"
zeXdI , "HN"
